### ISSUE TITLE
7-1 broken

### ISSUE THREAD

**Issue by pupsi-cola** [xzxADIxzx/Join-and-kill-em-together#40]
Installed the mod, played with my brother for a bit. A few days later I went through the 7th layer again (first time was without mod) (i wanted to P rank everything) and I couldn't complete 7-1 because I couldn't put the red skull down. I moved the mod folder so that bepinex doesn't see it, works perfectly. Another thing to note is that there's a second, red skull next to the first one that you can't grab

**Comment by pupsi-cola** [comment]
Just confirmed he is experiencing the same issue

**Comment by zUltimaPG3D** [comment]
the entirety of layer 7 will not work well, as nothing is synced correctly (except for maybe some non-act 3 enemies, but i don't know)

**Comment by zUltimaPG3D** [comment]
in simpler terms, layer 7 is broken if jaket is loaded

**Comment by DinoTrevor315** [comment]
Might as well add onto this. The 0-S door in 0-2 (The Meat Grinder) does not open even with the skull placed onto it's thing. So that's a thing

**Comment by Xioverze** [comment]
dont expect an update anytime soon, last time the dev touched this was 2 months ago. He might just not be working on github but nobody knows.

**Comment by PyroGabrielKotzell** [comment]
he's literally online on the discord server playing rn...
and yeah next update has a lotta stuff sooo...

**Comment by DinoTrevor315** [comment]
Updates take a really Long time to make. due to the amount of stuff added each update.

**Comment by zUltimaPG3D** [comment]
> dont expect an update anytime soon, last time the dev touched this was 2 months ago. He might just not be working on github but nobody knows.

"01/03/2024", one new commit, `add entity types for new enemies` in the `deep-refactoring` branch

**Comment by Xioverze** [comment]
> > dont expect an update anytime soon, last time the dev touched this was 2 months ago. He might just not be working on github but nobody knows.
> 
> "01/03/2024", one new commit, `add entity types for new enemies` in the `deep-refactoring` branch
my bad, thought that said it was 82 commits behind main

**Comment by xzxADIxzx** [comment]
This bug is currently fixed, just wait for the update
